content/_books/abstract-algebra.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Abstract Algebra
+updated_at: 2022-02-07
```

Comment:
I think it's a little silly to stick these on every single page. I originally added them to the important pages such as rules and info because that's what we already did in the discord channels.